spyder/widgets/reporterror.py

```diff
@@ -142,6 +142,7 @@ def __init__(self, parent=None):
         self.main_label.setOpenExternalLinks(True)
         self.main_label.setWordWrap(True)
         self.main_label.setAlignment(Qt.AlignJustify)
+        self.main_label.setStyleSheet('font-size: 11pt;')
```

Comment:
I don't like this. Text looks too big compared to the Description field. I think you should revert it.

Reply:
Okay, I see what you mean. I backed the size down to the size in the description field. However, its still larger than the original, which was actually way smaller than the text in the field, and so I could actually see how people could miss or gloss over it (I actually have a little trouble reading it myself even). I'll drop a screenshot; let me know what you think.